CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Dashboard/Health] Fix `an error as occured` in case of unknown `default_SR` (PR [#6508](https://github.com/vatesfr/xen-orchestra/pull/6508))
```

Comment:
```suggestion
- [Dashboard/Health] Fix `an error as occurred` in case of unknown default SR (PR [#6508](https://github.com/vatesfr/xen-orchestra/pull/6508))
```